integration_tests/reload_api_test.go

```diff
@@ -31,6 +32,39 @@ var _ = Describe("reload API endpoint", func() {
 			Expect(resp.StatusCode).To(Equal(405))
 			Expect(resp.Header.Get("Allow")).To(Equal("POST"))
 		})
+
+		Context("with a non-zero reload interval", func() {
+			newRouterPort := 7999
+			newApiPort := 8000
+			BeforeEach(func() {
+				err := startRouter(newRouterPort, newApiPort, map[string]string{"ROUTER_RELOAD_INTERVAL": "1s"})
+				Expect(err).To(BeNil())
+			})
+
+			AfterEach(func() {
+				stopRouter(newRouterPort)
+				// stopRouter doesn't actually seem to stop the router process for
+				// some reason. The only thing to do is to bump the port numbers and
+				// start yet another instance for the next test.
```

Comment:
It would be good to understand why this is an issue here.  It works correctly in [another test](https://github.com/alphagov/router/blob/master/integration_tests/proxy_function_test.go#L87-L102).  I'm worried that this could leave orphaned processes around on the CI machines, which will then interfere with future test runs etc.